git-visualize/src/app/app.module.ts

```diff
@@ -22,7 +22,10 @@ import {NgbModule} from '@ng-bootstrap/ng-bootstrap';
     FormsModule,
     NgbModule.forRoot()
   ],
-  providers: [],
-  bootstrap: [AppComponent]
+  providers: [CommunicateService],
+  bootstrap: [AppComponent],
+  schemas: [
+    CUSTOM_ELEMENTS_SCHEMA
```

Comment:
I think we should avoid capital words. `Custom_Elements_Schema` would be fine.

Reply:
Cannot understand it, sorry.
CUSTOM_ELEMENTS_SCHEMA is a library function